db/schema.sql

```diff
@@ -567,68 +568,70 @@ CREATE TABLE Users (
 /*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
 
 INSERT INTO MigrationVersions VALUES
-(1,'2024_11_11_160324_initial_schema','2025-04-30 12:54:12'),
```

Comment:
This big change is caused by me importing the production database locally, and as a result, the timestamps have all changed.